iree/compiler/Dialect/HAL/Target/ROCM/ROCMTargetUtils.cpp

```diff
@@ -0,0 +1,219 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iree/compiler/Dialect/HAL/Target/ROCM/ROCMTarget.h"
+#include "llvm/IR/Module.h"
+#include "llvm/IRReader/IRReader.h"
+#include "llvm/Linker/Linker.h"
+#include "llvm/Support/Program.h"
+#include "llvm/Transforms/IPO/Internalize.h"
+#include "mlir/Support/LogicalResult.h"
+
+// MLIR Support Files Header
```

Comment:
nit: comment doesn't seem completely accurate or very useful :). I would remove it.

Reply:
Done, I'll ping you again, once all the test pass :)

Reply:
Thanks! I enabled auto-merge once everything passes.